lib/myfinance/resources/payable_account.rb

```diff
@@ -9,26 +9,20 @@ module Resources
     class PayableAccount < FinancialAccount
 
       def pay(id, entity_id, params = {})
-        http.put(endpoints(id, entity_id, :pay), body: { payable_account: params }) do |response|
-          respond_with_object(response, 'payable_account')
-        end
+        request_and_build_response(:put, endpoint_for(id, entity_id, :pay), params)
       end
 
       def undo_payment(id, entity_id)
-        http.put(endpoints(id, entity_id, :undo_payment)) do |response|
-          respond_with_object(response, 'payable_account')
-        end
+        request_and_build_response(:put, endpoint_for(id, entity_id, :undo_payment))
       end
 
       private
 
-      def endpoints(id, entity_id, key)
-        {
-          create: '/entities/:entity_id/payable_accounts',
+      def endpoints
+        default_endpoints.merge({
           pay: '/entities/:entity_id/payable_accounts/:id/pay',
-          undo_payment: '/entities/:entity_id/payable_accounts/:id/undo_payment',
-          destroy: '/entities/:entity_id/payable_accounts/:id'
-        }[key].gsub(':entity_id', entity_id.to_s).gsub(':id', id.to_s)
+          undo_payment: '/entities/:entity_id/payable_accounts/:id/undo_payment'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.